src/firefly/js/templates/fireflyviewer/TriViewPanel.jsx

```diff
@@ -93,6 +95,7 @@ function RightSide({expanded, closeable, showXyPlots, showMeta, showFits, dataPr
     const cov= imagesWithCharts || coverageRight;
     const meta= imagesWithCharts && showMeta;
     const fits= imagesWithCharts && showFits;
+    const showPropertySheet = getAppOptions()?.table?.propertySheet === 'integrated';
```

Comment:
I think you should make `'integrated'` and `'popup'` constants in table util.

Reply:
Yes, that's better